.github/workflows/node-ci.yml

```diff
@@ -16,7 +16,14 @@ jobs:
         os: [ubuntu-latest]
         node-version: [16.x]
     steps:
+      # We don't need changesets for infrastructure-type files (tests, stories,
+      # etc).
       - uses: Khan/actions@check-for-changeset-v0
```

Comment:
Depends on https://github.com/Khan/actions/blob/check-for-changeset-v0.3.0/action.yml#L12 (which is published under `Khan/actions@check-for-changeset-v0`).